includes/class-wc-siftscience-admin.php

```diff
@@ -540,12 +564,13 @@ private function save_custom_fields() {
 				return;
 			}
 
-			foreach ( self::CUSTOM_FIELDS as $f ) {
-				if ( ! isset( $_REQUEST[ $f ] ) ) {
-					continue;
-				}
+			if ( isset( $_REQUEST['good_from'], $_REQUEST['good_to'], $_REQUEST['bad_from'], $_REQUEST['bad_to'] ) ) {
+				$good_from = sanitize_key( $_REQUEST['good_from'] );
+				$good_to   = sanitize_key( $_REQUEST['good_to'] );
+				$bad_from  = sanitize_key( $_REQUEST['bad_from'] );
+				$bad_to    = sanitize_key( $_REQUEST['bad_to'] );
 
-				update_option( $f, sanitize_key( $_REQUEST[ $f ] ) );
+				$this->options->set_order_auto_update_settings( $good_from, $good_to, $bad_from, $bad_to );
```

Comment:
As you want but i don't like storing sanitize_key( $_REQUEST['whatever'] );
just nest it with a large method call.